content/post/wxs-reflection-005.md

```diff
@@ -0,0 +1,54 @@
+optimistic about these, I did find the writing more enjoyable and that it had a
+bit more natural shape when I was directing it towards someone. So far I've sent
+one to Max and one to my friend Amanda. 
+
+I didn't copy the group on the one with Amanda, though I went back and forth on
+that. My idea with the correspondences is that I'll initiate them with folks
+where I feel an overlap in research interests that will benefit from the
+dialogue, which includes y'all in my cohort but I have a few folks outside that
+I'm also planning on writing. I found myself unsure how much I want to overload
+them with the whole theory of what I'm doing, explaining who these folks I'm
+CC'ing are, etc. On the other hand I think building up a shared corpus of ideas
+between us will eventually create a micro-culture that sustains its own meaning
+and shape within itself, and so adding new relevant conversations into that
+micro-culture will help it reach that auto-poietic tipping point.
+
+I also don't want to overload people. I've been feeling a bit of
+self-consciousness around how much content and energy I'm injecting into the
+system. I'm morally opposed to homework, and if this starts to feel like
+homework it hasn't quite achieved its goal. There's an interesting balance in
+trying to hold some amount of internally driven vision for what I'm trying to do
+while also being responsive to what is working for others and what isn't.
+
+I had a call with Jess recently and we talked about how this is what it's like
+with comedians, they're constantly trying out their new material on audiences,
+keeping the jokes that work and tweaking and dropping the ones that don't. A
+feedback loop in relationship with The Public. In the extreme this leads to
+exactly the kinds of audience capture dynamics which I want to avoid (and one
+reason that I'm not doing this work in the full chaos of Twitter or other
+"communities" of millions), but held in the right balance a comedian can have
+their own voice while also responding to what resonates.
+
+So, perhaps that's the strategy here, we're trying out various jokes: "let's do
+weekly reflections", "let's have a weekly meeting", "let's start
+correspondences", "let's start a reading group" and then seeing what makes the
+others laugh.
+
+There's a connection here to Serena's research of course, and 1rg itself. You're
+trying to hold some vision of what you're trying to do, while also responding to
+the needs and desires of the community. The art will be in threading between the
+absurd extremes "do whatever until you hit product-market fit", or full egoic
+"I'll do whatever the heck I want". And some measure of success will be when
+things happen without you being the source of energy, (like member--run side
+project nights). Similarly with the fellowship work, I will be excited when
+folks meet without me, or initiate a correspondence that I'm copied on but not
```

Comment:
A form of this may have happened yesterday when @mveytsman and I body sat down together with the intention to work on our fellowship reflections